.github/workflows/core-ci.yml

```
@@ -148,345 +135,154 @@ jobs:
-        with:
-          name: goblin-camp-${{ matrix.target }}-${{ github.sha }}
-          path: |
-            target/release/gc_cli
-          retention-days: 7
+      - name: Build CLI
+        run: cargo build --package gc_cli
 
-  test:
-    name: 🧪 Tests
+      - name: Validate Demos
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:2:9: Double quote to prevent globbing and word splitting

(shellcheck)